rules/no-negated-conjunction.ts

```
@@ -38,7 +37,7 @@ export default {
         })
 
         if (fixedExpression) {
-          let originalExpression = getNodeContent(node, context)
+          let originalExpression = context.sourceCode.getText(node)
```

ESLint

[error] Unsafe argument of type error typed assigned to a parameter of type `Node | undefined`.

(typescript/no-unsafe-argument)